gems/smithy/lib/smithy/anvil/client/templates/endpoint_parameters.erb

```diff
@@ -4,7 +4,14 @@
 
 module <%= namespace %>
   # Endpoint parameters used to resolve endpoints per request.
-  <%= documentation %>
+<% parameters.each do |param| -%>
+  # @!attribute <%= param.name %>
+<% if param.documentation -%>
+  #   <%= param.documentation %>
+<% end -%>
+  #
```

Comment:
Can probably remove this blank line for compactness.

Reply:
Blank lines are required for yard to render correctly (and in fact after testing, I needed to add more)

Reply:
Huh really? Maybe that's true for attribute but it shouldn't be for other tags.

Reply:
In v3 I don't see spaces between attribute and return.